packages/block-editor/src/components/link-control/index.js

```diff
@@ -26,16 +26,21 @@ import { useSelect } from '@wordpress/data';
 import LinkControlSettingsDrawer from './settings-drawer';
 import LinkControlSearchItem from './search-item';
 import LinkControlSearchInput from './search-input';
+import LinkControlSearchCreate from './search-create-button';
 
 function LinkControl( {
 	value,
 	settings,
 	onChange = noop,
 	showInitialSuggestions,
+	showCreatePages,
```

Comment:
Question: do we need `showCreatePages` if we already have to provide `createEmptyPage`? Seems like an overly verbose API.

Reply:
API has been removed.